synapse/models/infotech.py

```diff
@@ -222,6 +222,13 @@ def repr(self, valu):
     (50, 'red'),
 )
 
+suslevels = (
+    (10, 'benign'),
+    (20, 'unknown'),
+    (30, 'suspicious'),
+    (40, 'malicious'),
```

Comment:
for the VT result of `"undetected" (AV has no opinion about this file)`, would the verdict be `unknown` or would the prop be unset? The other verdicts of:
```
"type-unsupported" (AV can't analyse that file. Only returned in file analyses)`
"confirmed-timeout" (AV reached a timeout when analysing that file. Only returned in file analyses.)
"failure" (AV failed when analysing this file. Only returned in file analyses).
```
seem like they would fit under `unknown` but an undetected result seems like it could be slightly different?

Reply:
yep, sounds like "undetected" maps to "unknown" unless they're saying it means "benign"